Updated geometry for Presidio Heights neighbourhood

stepps00 opened the pull request:
fixes #127

stepps00 commented:
@nvkelso This pull request now contains edits to the North Beach and Presidio Heights neighborhoods, as well as a new record for Treasure Island. Requesting that this PR is merged to Master.

nvkelso commented on data/823/321/587/823321587.geojson:
```diff
@@ -0,0 +1,66 @@
+    "mz:min_zoom":15,
+    "mz:tier_locality":7,
+    "src:geom":"mapzen",
+    "src:geom_alt":[
```
There is no `geom_alt` for this feature, remove lines 22 to 24.

nvkelso commented:
The Financial District portion is covered in a separate PR: https://github.com/whosonfirst/whosonfirst-data/pull/283.

nvkelso commented:
Modulo alt geometry discussion in https://github.com/whosonfirst/whosonfirst-data/pull/283, 👍.

stepps00 commented:
@nvkelso - 1150db3 now includes the following:

* Removed `src:geom` from the new Treasure Island record
* Updated `src` fields, created a new zetashapes alt geojson, and Exportified all records for North Beach
* Updated `src` fields, created a new zetashapes alt geojson, and Exportified all records for Presidio Heights

nvkelso commented:
Almost there! But I'm seeing the new `mapzen` geom in the `zetashapes` alt-geom file. Let's go through how to dig out the old zetashape geom in the git history together on Thursday.

stepps00 commented:
Dug out the old zetashape geometries in the git history and replaced the `mapzen` geometries with the new new `zetashape` geometries in the alt-geom files.

The most recent [commit](5508bca) essentially replaced the alt-geom geometries with the geometries that were originally associated to `src: zetashapes`. All geometries should now associate to the appropriate source.

nvkelso commented:
@stepps00 Looks like the geometry in **85865911-alt-zetashapes.geojson** is corrupted? I can't preview it on Github.com, and won't load in GeoJSON.io, and fails jsonlint.com.

nvkelso commented:
(Everything else looks good, fix the above and send to Aaron.)

stepps00 commented:
[Most recent commit](f38ed41) takes care of the invalid geojson.
The spacing/indent issue was due to the geojson geometry `type` being listed as a `polygon`, rather than a `multipolygon`.

stepps00 commented:
Over to Aaron for validation.

thisisaaronland commented:
👍
